model/schema/gene.yaml

```diff
@@ -85,9 +86,61 @@ classes:
     is_a: GenomicEntityDTO
     description: Ingest class for genes
     slots:
-      - symbol
+      - gene_symbol_dto
+      - gene_full_name_dto
+      - gene_systematic_name_dto
+      - gene_synonym_dtos
       - gene_type_curie
 
+  GeneSymbolSlotAnnotation:
+    description: >-
+      The one current symbol for the gene.
+    is_a: NameSlotAnnotation
+    slots:
+      - single_gene
+    slot_usage:
+      single_gene:
+        required: true
+      name_type:
+        any_of:
+          - equals_string: nomenclature_synonym
+
+  GeneFullNameSlotAnnotation:
+    is_a: NameSlotAnnotation
+    description: >-
+      The one current full name for the gene.
+    slots:
+      - single_gene
+    slot_usage:
+      single_gene:
+        required: true
+      name_type:
+        any_of:
+          - equals_string: full_name
+
+  GeneSystematicNameSlotAnnotation:
+    is_a: NameSlotAnnotation
+    description: >-
+      The one current systematic name for the gene.
+    slots:
+      - single_gene
+    slot_usage:
+      single_gene:
+        required: true
+      name_type:
+        any_of:
+          - equals_string: systematic_name
```

Comment:
Like last several comments: I'm wondering: I know the name_type_name is a DTO class slot and so the range is "string" and therefore we can use the "equals_string" assertion, but I wonder if we can apply the same constraint for "name_type" where the range is actually 'VocabularyTerm'?

Reply:
Changed to note